### ISSUE TITLE
Update composer.json version

### ISSUE THREAD

**Issue by afbora** [floriankarsten/kirby-plausible#3]
composer.json version still point 0.1.4

**Comment by afbora** [comment]
0.1.5 tag still point 0.1.4. I think, you need publish new release with 0.1.6 😅

**Comment by iskrisis** [comment]
Yeah i've realized this just right now when i got notification for comment 😭

**Comment by afbora** [comment]
@iskrisis `0.1.16`? Do you mean `0.1.6`? 😄

**Comment by iskrisis** [comment]
now you are just putting salt to injury
